exec/java-exec/src/main/java/org/apache/drill/exec/planner/physical/LateralJoinPrel.java

```diff
@@ -48,7 +48,7 @@
 public class LateralJoinPrel extends DrillLateralJoinRelBase implements Prel {
 
 
-  protected LateralJoinPrel(RelOptCluster cluster, RelTraitSet traits, RelNode left, RelNode right, boolean excludeCorrelateCol,
+  public LateralJoinPrel(RelOptCluster cluster, RelTraitSet traits, RelNode left, RelNode right, boolean excludeCorrelateCol,
```

Comment:
I think there is no need to make this constructor public. In `LateralUnnestRowIDVisitor` may be used method `copy()` instead of using the constructor.